numpy/core/fromnumeric.py

```diff
@@ -691,8 +692,19 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     >>> x[np.argpartition(x, (1, 3))]
     array([1, 2, 3, 4])
 
+    >>> x = [3, 4, 2, 1]
+    >>> x[np.argpartition(x, 3)]
+    Traceback (most recent call last):
+    File "<stdin>", line 1, in <module>
+    TypeError: only integer arrays with one element can be converted to an index
+
```

Comment:
I think this is fine, but it will be helpful to add this to give a working example.

    >>>np.array(x)[np.argpartition(x, 3)]
    array([2, 1, 3, 4])